src/styles/styles.js

```diff
@@ -26,13 +26,11 @@ const picker = {
     fontFamily: fontFamily.EXP_NEUE,
     fontSize: variables.fontSizeNormal,
     lineHeight: variables.fontSizeNormalHeight,
-    paddingHorizontal: 11,
     paddingBottom: 8,
     paddingTop: 23,
+    paddingLeft: 0,
```

Comment:
Just curious why we are only using paddingLeft? Why not just set paddingHorizontal to 0?

Reply:
We could totally do `paddingHorizontal: 0` 
I'd just need to retest on everything.

Reply:
Well I'm mostly just trying to understand why we would do paddingLeft but not touch the paddingRight. Is there a good reason for that?

Reply:
No, there's not a good reason :/ 

 It was a two-stage change where I pulled out paddingHorizontal, and then later added paddingLeft. But I should have made it paddingHorizontal. 
 
 I just made this change and I'm retesting now.